clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/CreateServerGroupAtomicOperation.java

```diff
@@ -67,12 +70,16 @@
 public class CreateServerGroupAtomicOperation extends AbstractEcsAtomicOperation<CreateServerGroupDescription, DeploymentResult> {
 
   private static final String NECESSARY_TRUSTED_SERVICE = "ecs-tasks.amazonaws.com";
+  public static final String AWSVPC_NETWORK_MODE = "awsvpc";
 
   @Autowired
   EcsCloudMetricService ecsCloudMetricService;
   @Autowired
   IamPolicyReader iamPolicyReader;
 
+  @Autowired
+  SubnetSelector subnetSelector;
```

Comment:
Prefer autowired constructors over autowired properties.

Reply:
The atomic operations have autowired fields, since they require a constructor with a description.  AFAIK that's how it works for all atomic operations in Clouddriver.  I also prefer dependencies being injected in constructors.  However, given the `OperationsController` in clouddriver-web,  I don't think this approach is feasible.  What do you think?

Reply:
Hmm. Ok, that makes sense and is saddening. Thanks for looking into it.